test/e2e/simctl-e2e-specs.js

```diff
@@ -270,5 +272,30 @@ describe('simctl', function () {
         fullList.runtimes.filter((el) => el.identifier.includes('iOS')).length.should.be.above(0);
       });
     });
+
+    describe('getScreenshot', function() {
+      it('should get a base64 string', async function () {
+        const image = await simctl.getScreenshot();
+
+        // Use a rough regex to check if a returned value is base64 string
+        const regex = new RegExp(/[A-Za-z0-9+/=]/);
```

Comment:
you don't need to use an explicit RegExp constructor: const regex = `/^[A-Za-z0-9+/=]+$/`

Reply:
I think something like `Buffer.from(str, 'base64').toString('base64') === str` would better verify base64-encoded entity

Reply:
thank you, addressed as https://github.com/appium/node-simctl/pull/175/commits/b308bce0f39adce61e7eb105fbb086f049072755